### DISCUSSION TITLE
[1657630]Slime Rancher 2/史莱姆农场2

### DISCUSSION THREAD

**Discussion by maximun8686** [pjy612/SteamManifestCache#567] [Share]
`appid: 1657630`

keys:

```
1657631;a37c1409dc2c7880fc988d25d9fa333438bab756f17f7d0fc27015c8ef2faed3
```

manifest: [1657631_4393032796263052690.zip](https://github.com/pjy612/SteamManifestCache/files/14866828/1657631_4393032796263052690.zip)

code: 0C5D29462E8182A1

email: 2135869434@qq.com

**Comment by pjy612** [comment]
* #26 
* 调整密钥格式
